doc/userguide/install.rst

```diff
@@ -28,6 +28,9 @@ This will install Suricata into ``/usr/local/bin/``, use the default
 configuration in ``/usr/local/etc/suricata/`` and will output to
 ``/usr/local/var/log/suricata``
 
+You can obtain the source code for the latest release from our [downloads page](https://suricata.io/download/).
+The releases [on Github](https://github.com/OISF/suricata/releases/latest) _are not actual releases_:
+they are just a snapshot of the repository and do not include, for example, `./configure` and `suricata-update`.
```

Comment:
they do not include `libhtp`,  not `./configure`

Reply:
I'm not sure this is the place for mentioning GitHub as we don't mention it anywhere else in the user install docs.  But we do need mention of the proper download location.

Reply:
Or mention it without the link and less detail requiring Suricata developer knowledge: "Note that the release archives found on GitHub are snapshots of the git repo and are *not* installable by following this guide."

Reply:
Fair. What to do with the `./scripts/bundle/sh`   we have in this section, as pointed out by Brian?

Reply:
autogen.sh and bundle.sh should not be run when installing from release archives. They shouldn't hurt, but are for building from git, not our releases.